### PR TITLE
Add betterplace donation form

### LINKED ISSUE #218: Add Donation form
Code snippet from André:

```javascript
<script type="text/javascript">
  /* Configure at https://www.betterplace.org/de/projects/74137-unterstutze-die-lernwerkstatt-lernen-und-lernen-lassen-e-v/manage/iframe_donation_form/new */
  var _bp_iframe        = _bp_iframe || {};
  _bp_iframe.project_id = 74137; /* REQUIRED */
  _bp_iframe.lang       = 'de'; /* Language of the form */
  _bp_iframe.width = 600; /* Custom iframe-tag-width, integer */
  _bp_iframe.color = 'fd7222'; /* Button and banderole color, hex without "#" */
  _bp_iframe.background_color = 'ffffff'; /* Background-color, hex without "#" */
  _bp_iframe.default_amount = 10; /* Donation-amount, integer 1-99 */
  _bp_iframe.recurring_interval = 'monthly'; /* Interval for recurring donations, string out of ["single", "monthly", "quarter_yearly", "half_yearly", "yearly"] */
  _bp_iframe.bottom_logo = true;
  (function() {
    var bp = document.createElement('script'); bp.type = 'text/javascript'; bp.async = true;
    bp.src = 'https://betterplace-assets.betterplace.org/assets/load_donation_iframe.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(bp, s);
  })();
</script>
<div id="betterplace_donation_iframe" style="background: transparent url('https://www.betterplace.org/assets/new_spinner.gif') 275px 20px no-repeat;"><strong><a href="https://www.betterplace.org/de/donate/platform/projects/74137-unterstutze-die-lernwerkstatt-lernen-und-lernen-lassen-e-v">Jetzt Spenden für „Unterstütze Die Lernwerkstatt - lernen und lernen lassen e.V. “ bei unserem Partner betterplace.org</a></strong></div>
```

### PR DESCRIPTION
**Pull request by danielstreif**
**What**:
- embedded betterplace donation form on support page
- DE and EN version available

@pheeria did André have any specific requests where to place the form?

image

**Checklist**:

- [ ] Documentation added to the
      [README](https://github.com/Lernwerkstatt/site/blob/master/README.md)
- [ ] Tests
- [x] Ready to be merged
      

Fixes #218

### CODE REVIEW

File: src/views/support.hbs [hunk 1]
```diff
@@ -22,3 +22,27 @@
...
+
+<div class="container mb-3">
+  {{{__ "betterplace_text"}}}
+  <script type="text/javascript">
+    /* Configure at https://www.betterplace.org/de/projects/74137-unterstutze-die-lernwerkstatt-lernen-und-lernen-lassen-e-v/manage/iframe_donation_form/new */
+    var _bp_iframe = _bp_iframe || {};
+    _bp_iframe.project_id = 74137; /* REQUIRED */
+    _bp_iframe.lang = '{{{__ "locale"}}}'; /* Language of the form */
+    _bp_iframe.width = 600; /* Custom iframe-tag-width, integer */
+    _bp_iframe.color = 'fd7222'; /* Button and banderole color, hex without "#" */
```
Comment (pheeria) [inline src/views/support.hbs]: Can we use our primary color (it is 87C7C0 as of now) here?

File: src/views/support.hbs [hunk 2]
```diff
@@ -22,3 +22,27 @@
...
+    _bp_iframe.default_amount = 10; /* Donation-amount, integer 1-99 */
+    _bp_iframe.recurring_interval = 'monthly'; /* Interval for recurring donations, string out of ["single", "monthly", "quarter_yearly", "half_yearly", "yearly"] */
+    _bp_iframe.bottom_logo = true;
+    (function () {
+      var bp = document.createElement('script'); bp.type = 'text/javascript'; bp.async = true;
+      bp.src = 'https://betterplace-assets.betterplace.org/assets/load_donation_iframe.js';
+      var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(bp, s);
+    })();
+  </script>
+  <div class="mt-3" id="betterplace_donation_iframe"
```
Comment (pheeria) [inline src/views/support.hbs]: Could we maybe make the page have two columns instead of a long scroll?

File: src/views/support.hbs [hunk 3]
```diff
@@ -22,3 +22,27 @@
...
+    /* Configure at https://www.betterplace.org/de/projects/74137-unterstutze-die-lernwerkstatt-lernen-und-lernen-lassen-e-v/manage/iframe_donation_form/new */
+    var _bp_iframe = _bp_iframe || {};
+    _bp_iframe.project_id = 74137; /* REQUIRED */
+    _bp_iframe.lang = '{{{__ "locale"}}}'; /* Language of the form */
+    _bp_iframe.width = 600; /* Custom iframe-tag-width, integer */
+    _bp_iframe.color = 'fd7222'; /* Button and banderole color, hex without "#" */
+    _bp_iframe.background_color = 'ffffff'; /* Background-color, hex without "#" */
+    _bp_iframe.default_amount = 10; /* Donation-amount, integer 1-99 */
+    _bp_iframe.recurring_interval = 'monthly'; /* Interval for recurring donations, string out of ["single", "monthly", "quarter_yearly", "half_yearly", "yearly"] */
+    _bp_iframe.bottom_logo = true;
```
Comment (pheeria) [inline src/views/support.hbs]: Let's set this to `false` :)

File: src/views/support.hbs (hunk 2 above)
Comment (danielstreif) [reply src/views/support.hbs]: No problem :)

**Pull request merged**